tony-azkaban/src/main/java/com/linkedin/tony/azkaban/TensorFlowJob.java

```diff
@@ -104,30 +71,13 @@ public String getJavaClass() {
   protected String getJVMArguments() {
     String args = super.getJVMArguments();
 
-    String typeUserGlobalJVMArgs = getJobProps().getString(HadoopJobUtils.JOBTYPE_GLOBAL_JVM_ARGS, null);
-    if (typeUserGlobalJVMArgs != null) {
-      args += " " + typeUserGlobalJVMArgs;
-    }
-    String typeSysGlobalJVMArgs = getSysProps().getString(HadoopJobUtils.JOBTYPE_GLOBAL_JVM_ARGS, null);
-    if (typeSysGlobalJVMArgs != null) {
-      args += " " + typeSysGlobalJVMArgs;
-    }
-    String typeUserJVMArgs = getJobProps().getString(HadoopJobUtils.JOBTYPE_JVM_ARGS, null);
-    if (typeUserJVMArgs != null) {
```

Comment:
i remember we left some of these here for compatibility, make sure we don't need to force user to upgrade tony

Reply:
This is the exact same code as in HadoopJavaJob.getJVMArguments (which is already called via super.getJVMArguments() on line 72), so it should be safe to remove.

I also tested the jobtype changes in Azkaban (replaced the jar and reloaded the jobtype plugins) and it still worked fine.